tests/test_main.py

```diff
@@ -58,24 +59,23 @@ def test_cli_declare_acces() -> None:
 
 Options:
   --bas                           Use staging (bac à sable) environment
-  --client-secret TEXT            openid client secret  [required]
-  --client-id TEXT                openid client id  [required]
+  --client-secret TEXT            openid client secret
+  --client-id TEXT                openid client id
```

Comment:
I think you don't get `[required]` here because you ran test with CLIENT_ID & CLIENT_SECRET environment variables, but will fail in github tests.